aspnetcore/tutorials/first-web-api/sample/TodoApi/Startup.cs

```diff
@@ -28,10 +28,8 @@ public Startup(IHostingEnvironment env)
         // using Microsoft.EntityFrameworkCore;
         public void ConfigureServices(IServiceCollection services)
         {
-            // requires using Microsoft.EntityFrameworkCore;
             services.AddDbContext<TodoContext>(opt => opt.UseInMemoryDatabase());
 
-            // Add framework services.
             services.AddMvc();
             
             services.AddSingleton<ITodoRepository, TodoRepository>();
```

Comment:
@ajcvickers @divega  please review quick question
@tdykstra  asks:

Singletons are supposed to be threadsafe, but this class has a DbContext which is not threadsafe.